lib/ecto/adapters/sql/sandbox.ex

```diff
@@ -1,329 +1,299 @@
 defmodule Ecto.Adapters.SQL.Sandbox do
```

Comment:
Don't worry about logging here... at all. We are removing sandbox in favor of ownership for 2.0. This will all be gone.